tests/components/zeversolar/test_init.py

```diff
@@ -1,32 +1,65 @@
 """Test the init file code."""
 
+from unittest.mock import patch
+
 import pytest
+from zeversolar import ZeverSolarData
+from zeversolar.exceptions import ZeverSolarTimeout
 
 import homeassistant.components.zeversolar.__init__ as init
-from homeassistant.components.zeversolar.const import DOMAIN
-from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryNotReady
 
-from tests.common import MockConfigEntry, MockModule, mock_integration
+from tests.common import MockConfigEntry
+
+
+async def test_async_setup_entry_fails(
+    hass: HomeAssistant, config_entry: MockConfigEntry
+) -> None:
+    """Test to start the integration when inverter is offline (e.g. at night). Must raise a ConfigEntryNotReady error."""
+
+    config_entry.add_to_hass(hass)
+
+    with (
+        patch("zeversolar.ZeverSolarClient.get_data", side_effect=ZeverSolarTimeout),
+        pytest.raises(ConfigEntryNotReady),
+    ):
+        await init.async_setup_entry(hass, config_entry)
+
+
+async def test_load_entry(
+    hass: HomeAssistant, config_entry: MockConfigEntry, zeversolar_data: ZeverSolarData
+) -> None:
+    """Test loading the entry."""
+
+    with (
+        patch("homeassistant.components.zeversolar.PLATFORMS", [Platform.SENSOR]),
```

Comment:
Don't load any platform here...
```suggestion
        patch("homeassistant.components.zeversolar.PLATFORMS", []),
```